includes/class-wc-payments-woopay-button-handler.php

```diff
@@ -95,7 +95,7 @@ public function __construct( WC_Payments_Account $account, WC_Payment_Gateway_WC
 	 * @return bool
 	 */
 	public function is_woopay_enabled() {
-		return $this->is_woopay_eligible && $this->is_woopay_enabled && $this->is_woopay_express_button_enabled;
+		return apply_filters( 'wcpay_woopay_enabled', $this->is_woopay_eligible && $this->is_woopay_enabled && $this->is_woopay_express_button_enabled );
```

Comment:
(minor) consider adding a `@since` dockblock

```suggestion
		/**
		 * @since 9.5.0
		 */
		return apply_filters( 'wcpay_woopay_enabled', $this->is_woopay_eligible && $this->is_woopay_enabled && $this->is_woopay_express_button_enabled );
```